groups.go

```diff
@@ -803,3 +803,39 @@ func (s *GroupsService) DeleteGroupPushRule(gid interface{}, options ...RequestO
 
 	return s.client.Do(req, nil)
 }
+
+type ListProvisionedUsersOptions struct {
+	ListOptions
+	Username             *string    `url:"username,omitempty" json:"username,omitempty"`
```

Comment:
I think you need to run `go fmt` on this?

```diff
diff --git a/groups.go b/groups.go
index 3d24304..d819e54 100644
--- a/groups.go
+++ b/groups.go
@@ -806,12 +806,12 @@ func (s *GroupsService) DeleteGroupPushRule(gid interface{}, options ...RequestO
 
 type ListProvisionedUsersOptions struct {
 	ListOptions
-	Username             *string    `url:"username,omitempty" json:"username,omitempty"`
-	Search               *string    `url:"search,omitempty" json:"search,omitempty"`
-	Active               *bool `url:"active,omitempty" json:"active,omitempty"`
-	Blocked              *bool `url:"blocked,omitempty" json:"blocked,omitempty"`
-	CreatedAfter         *time.Time `url:"created_after,omitempty" json:"created_after,omitempty"`
-	CreatedBefore        *time.Time `url:"created_before,omitempty" json:"created_before,omitempty"`
+	Username      *string    `url:"username,omitempty" json:"username,omitempty"`
+	Search        *string    `url:"search,omitempty" json:"search,omitempty"`
+	Active        *bool      `url:"active,omitempty" json:"active,omitempty"`
+	Blocked       *bool      `url:"blocked,omitempty" json:"blocked,omitempty"`
+	CreatedAfter  *time.Time `url:"created_after,omitempty" json:"created_after,omitempty"`
+	CreatedBefore *time.Time `url:"created_before,omitempty" json:"created_before,omitempty"`
 }
 
 // ListProvisionedUsers gets a list of users provisioned by the given group.
```

Reply:
Thanks!